src/store/selectors/getFarmData.js

```diff
@@ -0,0 +1,6 @@
+import { FARM_ONBOARDING } from 'util/constants'
+
+export default function getFarmData (group) {
```

Comment:
this is not exactly a selector... but ok

Reply:
What's your name for a function that looks up data from the store?

Reply:
to me this is a utility function on the group model, so it could go in Group.js. A selector is a specific type of memoized function. And in react/redux code it usually specifically means one that pulls data from the redux state. but I also am ok leaving this it here